cmd/helm/installer/install.go

```diff
@@ -117,46 +118,46 @@ func generateLabels(labels map[string]string) map[string]string {
 	return labels
 }
 
-func generateDeployment(opts *Options) *extensions.Deployment {
+func generateDeployment(opts *Options) *v1beta1.Deployment {
 	labels := generateLabels(map[string]string{"name": "tiller"})
-	d := &extensions.Deployment{
+	d := &v1beta1.Deployment{
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: opts.Namespace,
 			Name:      deploymentName,
 			Labels:    labels,
 		},
-		Spec: extensions.DeploymentSpec{
-			Replicas: 1,
-			Template: api.PodTemplateSpec{
+		Spec: v1beta1.DeploymentSpec{
+			Replicas: util.Int32Ptr(1),
```

Comment:
Done (locally). I'll hold for a little while to see whether you find more points to address.